parser.hpp

```diff
@@ -174,6 +174,14 @@ namespace Sass {
       return peek< mx >(peek < css_comments >(start));
     }
 
+    // at-directives that aren't just prefixes
+    template <prelexer mx>
+    const char* peek_directive(const char* start = 0)
+    {
+      // now peek a token (skip comments first)
+      return peek< sequence< mx, negate< identifier > > >(start);
```

Comment:
I actually just saw that we already have the word boundary check implemented and in use for `extend` keywords. I just re-factored it a little bit and added a `word` prelexer to match a full word by a given kwd constant. Spec tests all pass and it should be more robust this way.
-> https://github.com/mgreter/libsass/commit/2a05583cde482c42bd78edf148cb4a53703e8d48